ts/LineRipple/index.tsx

```diff
@@ -0,0 +1,28 @@
+import {MDCLineRipple} from '@material/line-ripple';
+import autobind from 'autobind-decorator';
+import {h} from 'preact';
+import MaterialComponent from '../Base/MaterialComponent';
+
+export interface ILineRippleProps {}
+
+export interface ILineRippleState {}
+
+export class LineRipple extends MaterialComponent<
+  ILineRippleProps,
+  ILineRippleState
+> {
+  protected componentName = 'line-ripple';
+  protected mdcProps = [];
+  protected MDComponent: MDCLineRipple;
+
+  public componentDidMount() {
+    this.MDComponent = new MDCLineRipple(this.control);
```

Comment:
Missing `this.MDComponent.destroy()` in `componentWillUnmount()`. I know that it's not in the original code, but I assume it's because of an oversight.

Reply:
Maybe this should also be the case in MDComponent with the ripple thing

Reply:
Yes :)